checkipa

```diff
@@ -245,8 +263,10 @@ class IntegrityCheck(object):
         elif self.ipa_info['enterprise'] and not self.ipa_info['adhoc']:
             ipa_type = 'Enterprise'
         else:
-            ipa_type = 'unknown distribution type'
-            report_errors.append({'label': 'Distribution', 'description': 'unknown distribution type'})
+            if((len(provisioned_devices) == 0) and not(self.profile['Entitlements']['get-task-allow'])): 
+                ipa_type = 'AppStore Submission'
+            else:
+                report_errors.append({'label': 'Distribution', 'description': 'unknown distribution type'})
```

Comment:
within this block, ipa_type must be given a value (such as "unknown distribution type")